titus-api/src/main/java/com/netflix/titus/api/jobmanager/model/job/disruptionbudget/UnlimitedDisruptionBudgetRate.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2018 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.titus.api.jobmanager.model.job.disruptionbudget;
+
+public class UnlimitedDisruptionBudgetRate extends DisruptionBudgetRate {
```

Comment:
Singleton?

Reply:
I did not make these singleton to be consistent with the other builders. I would like to see it this way in case we add fields in the future.